state_machine/unit_tests/test_sun_position.py

```diff
@@ -0,0 +1,88 @@
+import unittest
+import sys
+from testutils import assert_vector_similar, timestamp
+
+sys.path.insert(0, './state_machine/applications/flight')
+
+from lib.sun_position import approx_sun_position_ECI
+
+class SunPosition(unittest.TestCase):
```

Comment:
My general style (and I think this is more widely accepted) is to prepend `Test` to the names of classes that inherit from `unittest.TestCase`. So this should be `TestSunPosition`.